src/realm/group_writer.hpp

```diff
@@ -181,6 +186,7 @@ class GroupWriter : public _impl::ArrayWriterBase {
     size_t m_evacuation_limit;
     int64_t m_backoff;
     size_t m_logical_size = 0;
+    bool m_final_step = false;
```

Comment:
This is a new variable, but it doesn't seem to be referenced anywhere in the new code?

Reply:
Right. That was for some other experiment.